Fix failing test for postgresql 12

```
  2) db
       postgresql
         given is already connected
           .getTableCreateScript
             should return table create script:

      AssertionError: expected 'CREATE TABLE public.users (\n  createdat date NULL,\n  role_id integer NULL,\n  password text NOT NULL,\n  email text NOT NULL,\n  username text NOT NULL,\n  id integer NOT NULL\n);\n\nALTER TABLE public.users ADD CONSTRAINT users_pkey PRIMARY KEY (id)' to deeply equal 'CREATE TABLE public.users (\n  id integer NOT NULL,\n  username text NOT NULL,\n  email text NOT NULL,\n  password text NOT NULL,\n  role_id integer NULL,\n  createdat date NULL\n);\n\nALTER TABLE public.users ADD CONSTRAINT users_pkey PRIMARY KEY (id)'
      + expected - actual

       CREATE TABLE public.users (
      -  createdat date NULL,
      +  id integer NOT NULL,
      +  username text NOT NULL,
      +  email text NOT NULL,
      +  password text NOT NULL,
         role_id integer NULL,
      -  password text NOT NULL,
      -  email text NOT NULL,
      -  username text NOT NULL,
      -  id integer NOT NULL
      +  createdat date NULL
       );
       
       ALTER TABLE public.users ADD CONSTRAINT users_pkey PRIMARY KEY (id)
      
      at Context.<anonymous> (spec/db.spec.js:319:39)
      at Generator.next (<anonymous>)
      at step (spec/db.spec.js:31:191)
      at /home/runner/work/sqlectron-core/sqlectron-core/spec/db.spec.js:31:361
      at process._tickCallback (internal/process/next_tick.js:68:7)
```

--- a/.github/workflows/test.yml
+++ b/.github/workflows/test.yml
@@ -28,7 +28,7 @@ jobs:
           - node-version: 10.x
             mysql-version: 5.7
             cassandra-version: 3.7
-            postgres-version: 9
+            postgres-version: 12
           - node-version: 12.x
             mysql-version: 5.7
             cassandra-version: 3.7

--- a/src/db/clients/postgresql.js
+++ b/src/db/clients/postgresql.js
@@ -356,6 +356,7 @@ export async function getTableCreateScript(conn, table, schema) {
       array_to_string(
         array_agg(
           '  ' || quote_ident(tabdef.column_name) || ' ' ||  tabdef.type || ' '|| tabdef.not_null
+          ORDER BY tabdef.column_order ASC
         )
         , E',\n'
       ) || E'\n);\n' ||
@@ -373,7 +374,8 @@ export async function getTableCreateScript(conn, table, schema) {
           WHEN a.attnotnull THEN 'NOT NULL'
         ELSE 'NULL'
         END AS not_null,
-        n.nspname as schema_name
+        n.nspname as schema_name,
+        a.attnum as column_order
       FROM pg_class c,
        pg_attribute a,
        pg_type t,

